css/src/atomics/colors.scss

```diff
@@ -1,6 +1,6 @@
 // Framework colors
 
-@each $name, $color-set in $colors {
+@each $name, $color-set in $framework-colors {
```

Comment:
I don't mind this change, but was there a particular reason for it? If not, can we revert?

Reply:
`$colors: $framework-colors` was problematic because the compiled value provided by Sass-export was a string of the values.

`"$colors": "(\"primary\": (var(--theme-primary-base), var(--theme-primary-background), ...), \"secondary\": (var(--theme-secondary-base), var(--theme-secondary-background), var(--theme-secondary-dark),..."` 

I can try to parse/handle this string if we need to keep `$colors: $framework-colors`. I just thought that it'd be simpler to use `$framework-colors` directly instead of nesting it within another token.

Unlike simpler tokens, it didn't provide the map value, which made it easy to form the desired object.  

`"colors": {
	"primary": "var(--theme-primary-base), var(--theme-primary-background), ...
	"secondary": "var(--theme-secondary-base), var(--theme-secondary-background), ...
}`

Reply:
If memory serves (it so rarely does), I included $colors because it was the variable that made the most sense, but didn't remove framework colors because it's used in docs.

Reply:
Update: as usually my memory can't be trusted. I think we used to mush brand and framework colors together, but it seems we're no longer doing that in docs.

Reply:
In short, I think we can remove the line `$colors: $framework-colors`, and just rename `$framework-colors` to `$colors`.

Reply:
Yes, I couldn't find where the combining was actually happening, hence the changes. We can go with the renaming. No parsing will be needed as long as the token structure is `$tokenName: value` and not `$tokenName: anotherTokenName -> an object`.

Reply:
Renamed to `$framework-colors` to `$colors`.